neural_modelling/src/neuron/neuron.c

```diff
@@ -61,11 +61,12 @@ static address_t current_source_address;
 //! parameters that reside in the neuron_parameter_data_region
 struct neuron_parameters {
     uint32_t has_key;
-    uint32_t transmission_key;
     uint32_t n_neurons_to_simulate;
     uint32_t n_neurons_peak;
     uint32_t n_synapse_types;
     uint32_t ring_buffer_shifts[];
+    // Also here is:
```

Comment:
Perhaps a better comment would be "Following this struct is:" as it's not really in this struct.

Reply:
Fixed